common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -906,6 +906,30 @@ def test_new_account_registration_assigns_distinct_username_on_collision(self):
         distinct_username = pipeline.get(request)['kwargs']['username']
         assert original_username != distinct_username
 
+    @django_utils.override_settings(ENABLE_REQUIRE_THIRD_PARTY_AUTH=True)
```

Comment:
This test is passing even with `ENABLE_REQUIRE_THIRD_PARTY_AUTH=False`, shouldn’t it fail without this setting enabled?

Reply:
😯 it should indeed fail without this setting enabled... I'll look into it. Sorry about that!